frontend/common/src/fakeData/fakePoolCandidates.ts

```diff
@@ -10,28 +10,32 @@ import {
   Pool,
   User,
   Applicant,
+  PoolAdvertisement,
 } from "../api/generated";
 import fakeClassifications from "./fakeClassifications";
 import fakePools from "./fakePools";
 import fakeUsers from "./fakeUsers";
+import fakePoolAdvertisements from "./fakePoolAdvertisements";
 
 const generatePoolCandidate = (
   pools: Pool[],
+  poolAdvertisements: PoolAdvertisement[],
   users: User[],
   classifications: Classification[],
 ): PoolCandidate => {
   faker.setLocale("en");
   return {
     id: faker.datatype.uuid(),
     pool: faker.helpers.arrayElement(pools),
+    poolAdvertisement: faker.helpers.arrayElement(poolAdvertisements),
     expectedClassifications:
       faker.helpers.arrayElements<Classification>(classifications),
     user: faker.helpers.arrayElement<User>(users) as Applicant,
     cmoIdentifier: faker.helpers.slugify(
       faker.lorem.words(faker.datatype.number({ min: 1, max: 3 })),
     ),
     expiryDate: faker.date
-      .between("2100-01-01", "2100-12-31")
+      .between("2022-01-01", "2100-12-31")
```

Comment:
not sure that up till 2100-12-31 is very useful now, as we'll basically never have expired candidates then
maybe thru 2023-12-31 instead?

Reply:
hang on, that does create another issue of changing snapshots as some switch to expired say a month later
maybe recycling the constants from constants.php
```
        'far_future_date' => '2050-01-01',
        'past_date' => '2020-01-01',
```

then do a not-so-elegant method to weigh the odds towards unexpired by

```
expiryDate: faker.helpers.arrayElement([future, future, future, past]),
```

Reply:
Great suggestion, updated with [update fake expiry date](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3819/commits/c5a229f095d4c585efe692c17c0d5fbfaccc3dc7)

Reply:
sorry, but it looks like that commit updated submittedAt instead of the expiry date? 

![image](https://user-images.githubusercontent.com/40485260/188183682-9653d5ac-1c0a-44ac-87e5-4e5b88b1e423.png)

Reply:
YIKES, good catch 😓  I have updated both with [fix fake pool candidate dates](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3819/commits/b2ccd03223eeaf287690d57fbee118849240a202)

> NOTE: I set submittedAt to `FAR_PAST_DATE` since it should always be in the past.